### PR TITLE
modify routine sensor1 -> sensor_base with parallel loop inside

### PR DESCRIPTION
**Pull request by MaciekWronski**
#### Checklist

- [x] The title of the PR is reviewed
- [x] There is no text before the checklist section
- [x] The following two sections are filled (or deleted)
- [x] This PR has been previewed 

#### Description of the feature or the bug

no change

#### Description of the changes

code clean after introducing new sensor structure. sensor1() routine becomes sensor_base() with parallel loop inside.
Sensor TIME is put in a separate routine like all other sensors

### CODE REVIEW

File: engine/source/tools/sensor/sensor_base.F [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+      INTEGER TYP,ISENS
+C-----------------------------------------------
+C   D e s c r i p t i o n
+C-----------------------------------------------
+C     this routine checks activation condition of all basic sensor types
+C=======================================================================
+c     Parallel loop over sensor list
+c--------------------------------------------------
+
+!$OMP PARALLEL DO PRIVATE(ISENS)
```
Comment (laurent-altr) [inline engine/source/tools/sensor/sensor_base.F:103]: Bug here: TYP shouod be private too

**Review by laurent-altr — CHANGES_REQUESTED** [review]
Please fix the OpenMP issue and review the list of tasks.

File: engine/source/tools/sensor/sensor_base.F (hunk 1 above)
Comment (MaciekWronski) [reply engine/source/tools/sensor/sensor_base.F]: merci, je viens de corriger

Comment (laurent-altr) [reply engine/source/tools/sensor/sensor_base.F]: Hi @MaciekWronski, is this PR ready?

**Review by laurent-altr — APPROVED** [review]

**Pull request merged**